### PR TITLE
[KOGITO-706] scorecard adjustments for 0.6.0 release

### PR DESCRIPTION
**Pull request by ricardozanini**
See:
https://issues.redhat.com/browse/KOGITO-706

Small adjustments to push 0.6.0 to OperatorHub

@radtriste @sutaakar could you guys please review?
A full working sample can be found in here: https://gist.github.com/ricardozanini/20cbc1b7768610cd86a6261f49a955b2

I also added a new channel, `dev-preview` to release 0.7.0-cr1 in two weeks, this way we can have quick milestones during development and treat bugs as soon as they happen.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
   # details about the connected infinispan
   infinispan:
     # let's leave this burden to KogitoInfra CR to deploy a new Infinispan instance for us
-    useKogitoInfra: true
+    useKogitoInfra: false
```
Comment (sutaakar) [inline deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml]: What is the reason for this change?

Comment (ricardozanini) [reply deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml]: To run the integration tests without the need of having Strimzi and Infinispan in the namespace. The scorecard smoke tests doesn't rely on OLM, thus won't install Strimzi or Infinispan for us, failing the use case.

Comment (sutaakar) [reply deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml]: ok, thanks.
